server/routes/coursesRouter.js

```diff
@@ -17,6 +17,10 @@ coursesRouter.post("/courses-details", async (req, res) => {
   }
 });
 
+coursesRouter.get("/leaderboard", async (_, res) => {
+  res.status(200).json(await db.getTop5BestReviewScores())
```

Comment:
This should be in a try catch that returns a status 400 if it fails to fetch from the db